src/components/organisms/settings/user/organizations/warningDeleteModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { useTranslation } from "react-i18next";
+
+import { ModalName } from "@enums/components";
+
+import { useModalStore } from "@store";
+
+import { Button } from "@components/atoms";
+import { Modal } from "@components/molecules";
+
+export const WarningDeleteOrganizationModal = () => {
+	const { t } = useTranslation("settings", { keyPrefix: "organization.modal" });
+	const { closeModal } = useModalStore();
+	const organization = useModalStore((state) => state.data) as { name: string };
```

Comment:
Why not:
```suggestion
	const name = useModalStore((state) => state.data);
```

and then in the JSX:
```
				<p>{t("organizationDeleteWarning", { name })}</p>
```

Reply:
we can change, But for call it we need `<p>{t("organizationDeleteWarning", { name: name.name })}</p>`. So better my solution